feature-wallet-api/src/main/java/jp/co/soramitsu/feature_wallet_api/data/cache/AssetCache.kt

```diff
@@ -98,4 +99,18 @@ class AssetCache(
             assetDao.updateAssets(onlyUpdates)
         }
     }
+
+    suspend fun updateAssetNotNeed(chainId: ChainId, metaId: Long, symbol: String) {
```

Comment:
Is that the function you moved to UseCase? Is it used anywhere?

Reply:
this fun is called in feature wallet and use case used in feature account